README.md

```diff
@@ -1,25 +1,31 @@
 Alerts v2
 =========
 
-Alerts for nib pages.
+Alerts for nib pages. There are two types of alerts - with and without a title.
 
-    <div class="v2-alert v2-alert--error">
-        <div class="v2-alert__title">Message Title</div>
-        Message content
+Alerts with a title:
+
+    <div class="alert alert--error">
+      <div class="alert__title">We're sorry, due to too many invalid login attempts your account is locked</div>
+      <div class="alert--message">You will be able to try again in 00:30 minutes.</div>
```

Comment:
`message` would be an element rather than a modifier too. http://getbem.com/naming/